explore_warming.ipynb

```diff
@@ -85,9 +84,129 @@
    },
    "outputs": [],
    "source": [
-    "from climakitae.explore import warming_levels\n",
     "wl = warming_levels()\n",
-    "wl.show()"
+    "wl.choose_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6853f5e9-8084-4f69-b8c4-acc36d615278",
+   "metadata": {},
+   "source": [
+    "### Now retrieve and process the data\n",
+    "The calculate step may take a while to complete. Selecting statistical downscaling will take longer because there are more simulations to work with, and they are at the highest spatial resolution (~10-15min)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f0f14a5e-e07c-4111-892f-2afbd4f314e8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "wl.calculate()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ec9685c-939c-437b-bcf5-b0512e1a63f0",
+   "metadata": {},
+   "source": [
+    "### Next visualize the regional response at a series of global warming levels.\n",
+    "Use the drop down menu to visualize when a specified global warming level in reached for a scenario of interest. Scenarios shown are Shared Socioeconomic Pathways ([SSPs](https://www.sciencedirect.com/science/article/pii/S0959378016300681)): ranging from low (SSP 1-1.9) to high (SSP 5-8.5) emissions trajectories. \n",
+    "\n",
+    "To learn more about the data available on the Analytics Engine, [see our data catalog](https://analytics.cal-adapt.org/data/). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b538032f-63d9-4cdd-8534-c7acb1252b33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wl.visualize()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "183deba2-9aca-45ee-bed8-8ae90c4db735",
+   "metadata": {},
+   "source": [
+    "wl.visualize() step, above, is optional if you want to go directly to extracting the data."
```

Comment:
nitpick (not critical to change but could be nice): Re-word to "The `wl.visualize()` step above is optional if you want go directly to extracting the data."